<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_code</class>
 <widget class="QDialog" name="change_code">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#change_code{
image: url(:/new/prefix1/v2-c2a88b13d61d4bedfd45ec2438839683_720w.jpg);
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>新密码确认：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>（密码以字母开头，长度在6~18之间，只能包含字母、数字和下划线）</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>change_code</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_4</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
